code/style.css

```diff
@@ -1,4 +1,228 @@
-html {
-  background: #0025ff;
+body {
+  background:#000;
   color: #fff;
+  margin: 000;
 }
+
+h1{
+  font-family: 'Playfair Display', sans-serif;
+  color: #fff;
+  font-weight: 100;
+  font-size: 50px;
+  margin-top: -12px;
```

Comment:
Try not to use a negative value in css. But in some cases you can't help it. But just a small reminder!